client/src/views/login/Login.js

```diff
@@ -1,17 +1,32 @@
-import React from 'react';
-import { useHistory } from 'react-router-dom';
+import React, { useEffect, useState } from 'react';
+import { useParams, useHistory } from 'react-router-dom';
 import ValidationForm from '../../components/validation/ValidationForm';
 
 function Login() {
+  const { comingFrom } = useParams();
   const history = useHistory();
-  
+  const [showError, setShowError] = useState(false);
+
   const handleValidate = () => {
     history.push('/');
-  }
+  };
+
+  const fetchError = () => {
+    let params = new URLSearchParams('route=' + comingFrom);
+    setShowError(params.get('route') == 'custInfo');
+  };
+
+  useEffect(fetchError, [comingFrom]);
 
   return (
     <div>
-      <h1>Enter your credentials to login:</h1>
+      <h1>Customer Login</h1>
+      {showError ? (
+        <h6>Credentials required for access to customer info page</h6>
+      ) : (
+        ''
+      )}
+
```

Comment:
Minor: if you only want a conditional render with no "else", like it is the case here, you don't have to use the ternary operation. You can use ```&&```, as per [the doc](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator).